autofit/graphical/factor_graphs/jacobians.py

```diff
@@ -52,16 +52,17 @@ def __init__(
             *,
             name="",
             vectorised=False,
-            is_scalar=False,
+            plates: Tuple[Plate, ...] = (),
             **kwargs: Variable
     ):
         self.vectorised = vectorised
-        self.is_scalar = is_scalar
+        self.is_scalar = bool(plates)
```

Comment:
Surely this evaluates to True if len(plates) > 0 which I would have thought is the opposite of what you want?

Reply:
Yeah, that's a mistake